closeio_api/__init__.py

```diff
@@ -50,59 +59,71 @@ def __init__(self, base_url, api_key=None, tz_offset=None, max_retries=5,
 
         self.session = requests.Session()
         if api_key:
-            self.session.auth = (api_key, '')
-
-        self.session.headers.update({
-            'User-Agent': 'Close/{} python ({})'.format(
-                __version__,
-                requests.utils.default_user_agent()
-            ),
-            'X-TZ-Offset': self.tz_offset
-        })
-
-    def _prepare_request(self, method_name, endpoint, api_key=None, data=None,
-                         debug=False, **kwargs):
+            self.session.auth = (api_key, "")
+
+        self.session.headers.update(
+            {
+                "User-Agent": "Close/{} python ({})".format(
+                    __version__, requests.utils.default_user_agent()
+                ),
+                "X-TZ-Offset": self.tz_offset,
+            }
+        )
+
+    def _prepare_request(
+        self,
+        method_name: RequestMethod,
+        endpoint: str,
+        api_key: Optional[str] = None,
+        data: Optional[Dict[Any, Any]] = None,
+        debug: bool = False,
+        **kwargs: Any
+    ) -> requests.PreparedRequest:
         """Construct and return a requests.Request object based on
         provided parameters.
         """
         if api_key:
-            auth = (api_key, '')
+            auth = (api_key, "")
```

Comment:
I'd prefer shipping changes to formatting separately from functional ones - putting them together makes actual changes really hard to review.

Reply:
I will make a different PR for `isort` and `black` after having specific requirements for dev/test dependencies